pkg/devfile/generator/utils_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
+var isTrue bool = true
```

Comment:
suggest to use `const` instead of `var`. Same for all other `isTrue`/`isFalse`
```suggestion
const isTrue bool = true
```

Reply:
Constants are not addressable types: https://golang.org/ref/spec#Address_operators so this will cause a compile error when assigning the address of the constant